alphastats/gui/utils/ollama_utils.py

```diff
@@ -179,24 +181,39 @@ def execute_function(
             If the function is not implemented or the dataset is not available
         """
         try:
-            if function_name == "get_gene_function":
-                # TODO log whats going on
```

Comment:
What happened to this TODO?

Reply:
there's now the option to show the whole conversation